apps/core/src/auth/cognito-jwt-auth.guard.ts

```diff
@@ -66,12 +78,9 @@ export class CognitoJwtAuthGuard implements CanActivate {
     }
   }
 
-  private getBearerToken(context: ExecutionContext): string {
-    const { headers: { authorization = '' } = { authorization: '' } } = context
-      .switchToHttp()
-      .getRequest<{
-        headers: { authorization: string | undefined } | undefined;
-      }>();
+  private getBearerToken(httpRequest: HttpRequest): string {
+    const { headers: { authorization = '' } = { authorization: '' } } =
```

Comment:
const { headers: { authorization } } = httpRequest;
return (authorization ?? '').replace(/^Bearer /, '');

might be a better way ?

Reply:
in `const { headers: { authorization } } = httpRequest;`, the `headers` might be undefined so cannot do it that way.

updated to the following for simplicity:
```
    const authorization = httpRequest.headers?.authorization ?? '';

    return authorization.replace(/^Bearer /, '');
```